met-api/src/met_api/services/membership_service.py

```diff
@@ -17,44 +19,77 @@ class MembershipService:
     def create_membership(engagement_id, request_json: dict):
         """Create membership."""
         user_id = request_json.get('user_id')
-        user: StaffUserModel = StaffUserModel.get_user_by_external_id(user_id)
-        if not user:
+        user_db: StaffUserModel = StaffUserModel.get_user_by_external_id(user_id)
```

Comment:
lol..i like the other way.. this is user ;not user_db
the other guy which gets serialised , is something like user_data or something of that sort..
he he..dont change it;but persoanl preference

Reply:
reverted the change